andela/casablanca-cfh
# #156944014 Fix The Failing Chat Functionality on Heroku

#### What does this PR do?

- rename fireBase.js to firebase.js

#### Description of Task to be completed?
Fix Chat Feature that is failing on Heroku
#### How should this be manually tested?

- visit the deployed app `https://casablanca-cfh-staging.herokuapp.com/#!/`
- start a game session
- open the chat window
- you should be able to chat with the other players in the game
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?(if applicable)
#156944014
